### PR TITLE
Fix/arrows render as emoji

### PR DESCRIPTION
**Pull request by JakubSerafin**
Arrow icon in some browser was rendered as a emoji 
obraz
This fix forced it to render as a text, and add some more gap between text and icon 
obraz

### CODE REVIEW

**Comment by dsagal** [comment]
@anaisconce , design-wise, is this fine, or should we use the Expand icon that we use elsewhere in Grist? 

Expand

To use an icon, Jakub would need to do a bit more to allow overriding the weasel library's default HTML element for this, since the library should need to know about Grist icons. It should not be difficult though.

**Comment by JakubSerafin** [comment]
Annis told me on slack that Expand icon is prefer here. I will update this PR

**Comment by berhalak** [comment]
- Adding a way to override expand icon, either globally (by css) or individually per sub menu.
- Updating the default style

image

File: lib/menu.ts [hunk 1]
```diff
@@ -412,7 +412,13 @@ export function menuItemSubmenu(
       (yesNo: boolean) => yesNo || ctl.close()),
 
     // Clicks that open a submenu should not cause parent menu to close.
-    dom.on('click', (ev) => { ev.stopPropagation(); }),
+    dom.on('click', (ev, elem) => {
+      if (options.action && !elem.classList.contains('disabled')) {
+        options.action(ev.target as HTMLElement);
```
Comment (georgegevoian) [inline lib/menu.ts:417]: What do you think about also passing in `ev` here? Might save us from a future PR if we need to keep the parent menu open when `action` is called.

File: test/fixtures/menu.ts [hunk 2]
```diff
@@ -334,3 +337,12 @@ const cssPopupContent = styled('div', `
 document.addEventListener('DOMContentLoaded', () => {
   document.body.appendChild(setupTest());
 });
+
+const cssOverride = styled('div', `
```
Comment (georgegevoian) [inline test/fixtures/menu.ts:341]: Is this unused?

**Review by georgegevoian — COMMENTED** [review]
Thanks. Just a few small comments.

File: test/fixtures/menu.ts (hunk 2 above)
Comment (berhalak) [reply test/fixtures/menu.ts:341]: No, only for manual test, I'll add a way to toggle it through browser.

File: package.json [hunk 3]
```diff
@@ -7,8 +7,8 @@
   "scripts": {
     "build": "tsc",
     "test": "MOCHA_WEBDRIVER_STACKTRACES=1 mocha 'test/browser/**/*.{js,ts}'",
-    "test-debug": "MOCHA_WEBDRIVER_STACKTRACES=1 mocha --bail test/browser/**/*.{js,ts} --no-exit",
-    "test-manual": "webpack-dev-server --config test/fixtures/webpack.config.js",
+    "test-debug": "NODE_OPTIONS=--openssl-legacy-provider MOCHA_WEBDRIVER_STACKTRACES=1 mocha --bail test/browser/**/*.{js,ts} --no-exit",
```
Comment (georgegevoian) [inline package.json:10]: I wonder if upgrading Webpack will resolve this. Anyway, ok to worry about later since it's a separate issue.

**Review by georgegevoian — APPROVED** [review]
Thanks!

**Pull request merged**